test/propagation/http_propagator_test.rb

```diff
@@ -8,18 +8,20 @@ def test_inject!
 
     tracer.trace('caller') do |span|
       env = { 'something' => 'alien' }
-      Datadog::HTTPPropagator.inject!(span, env)
+      Datadog::HTTPPropagator.inject!(span.context, env)
```

Comment:
Yes, `span.context`, perfect.

Reply:
yes always use the context attached to the `Span`